data/sql/updates/pending_db_world/update-sai-ssc-trash.sql

```diff
@@ -0,0 +1,120 @@
+(21301, 0, 0, 0, 0, 0, 100, 0, 6850, 12350, 19400, 27400, 0, 0, 11, 38591, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 0, 'Coilfang Shatterer - In Combat - Cast Shatter Armor'),-- fully sniffed
+(21301, 0, 1, 0, 4, 0, 100, 0, 0, 0, 0, 0, 0, 0, 39, 20, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Coilfang Shatterer - On Aggro - Call For Help'),
+(21339, 0, 0, 0, 0, 0, 100, 0, 9700, 22400, 13350, 40050, 0, 0, 11, 38491, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 0, 'Coilfang Hate-Screamer - In Combat - Cast Silence'),-- fully sniffed
+(21339, 0, 1, 0, 0, 0, 100, 0, 3600, 10900, 4850, 12950, 0, 0, 11, 38496, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 0, 'Coilfang Hate-Screamer - In Combat - Cast Sonic Scream'),-- fully sniffed
+(21806, 0, 0, 0, 0, 0, 100, 0, 6050, 9250, 8500, 11600, 0, 0, 11, 37531, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 0, 'Greyheart Spellbinder - In Combat - Cast Mind Blast'),-- fully sniffed
+(21806, 0, 1, 0, 105, 0, 100, 0, 6050, 6050, 8000, 10000, 0, 45, 11, 39076, 0, 0, 0, 0, 0, 7, 0, 0, 0, 0, 0, 0, 0, 0, 'Greyheart Spellbinder - In Combat - Cast Spell Shock'),-- repeat timer not sniffed
+(21806, 0, 2, 0, 4, 0, 100, 0, 0, 0, 0, 0, 0, 0, 39, 20, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Greyheart Spellbinder - On Aggro - Call For Help'),
+(21806, 0, 3, 0, 1, 0, 100, 1, 1000, 1000, 0, 0, 0, 0, 11, 37626, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Greyheart Spellbinder - Out of Combat - Cast Green Beam'),
+(21863, 0, 0, 0, 0, 0, 100, 0, 9650, 15750, 12150, 15750, 0, 0, 11, 38650, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Serpentshrine Lurker - In Combat - Cast Rancid Mushroom'),-- fully sniffed
+(21863, 0, 1, 0, 0, 0, 100, 0, 12150, 18150, 15000, 22000, 0, 0, 11, 38655, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Serpentshrine Lurker - In Combat - Cast Poison Bolt Volley'),-- repeat timer not sniffed
+(21865, 0, 0, 0, 0, 0, 100, 0, 1200, 9500, 3200, 11400, 0, 0, 11, 37770, 64, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 0, 'Coilfang Ambusher - In Combat - Cast Shoot'),-- fully sniffed
+(21865, 0, 1, 0, 0, 0, 100, 0, 7250, 14850, 12100, 16000, 0, 0, 11, 37790, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 0, 'Coilfang Ambusher - In Combat - Cast Spread Shot'),-- fully sniffed
+(21873, 0, 0, 0, 0, 0, 100, 0, 1000, 5000, 9000, 12000, 0, 0, 11, 28168, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 0, 'Coilfang Guardian - In Combat - Cast Arcing Smash'),-- not sniffed correctly
+(21873, 0, 1, 0, 0, 0, 100, 0, 8000, 12000, 14000, 18000, 0, 0, 11, 9080, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 0, 'Coilfang Guardian - In Combat - Cast Hamstring'),-- not sniffed but weird ID
+(21913, 0, 0, 0, 0, 0, 100, 0, 6000, 10300, 8450, 10750, 0, 0, 11, 39848, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Water Globule - In Combat - Cast Water Globule New Target'),-- fully sniffed
+(21913, 0, 1, 0, 0, 0, 100, 0, 6000, 42700, 4850, 14550, 0, 0, 11, 37871, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 0, 'Water Globule - In Combat - Cast Freeze'),-- fully sniffed
```

Comment:
These behave different, check the boss tactics, they aren't cast in a timer, at least freeze isn't

Reply:
might also want to move these to cpp. I think mangos does that too because no AI entry

Reply:
![image](https://github.com/azerothcore/azerothcore-wotlk/assets/83884799/9be3934d-a841-452e-b127-8c2da04588fb)
bingo